app/proguard-rules.txt

```diff
@@ -68,49 +79,17 @@
 -keep interface org.joda.time.** { *; }
 
 # ButterKnife
+-keep class butterknife.** { *; }
 -dontwarn butterknife.internal.**
 -keep class **$$ViewInjector { *; }
--keepnames class * { @butterknife.InjectView *;}
-
-#ACRA specifics
-# Restore some Source file names and restore approximate line numbers in the stack traces,
-# otherwise the stack traces are pretty useless
--keepattributes SourceFile,LineNumberTable
-
-# ACRA needs "annotations" so add this...
-# Note: This may already be defined in the default "proguard-android-optimize.txt"
-# file in the SDK. If it is, then you don't need to duplicate it. See your
-# "project.properties" file to get the path to the default "proguard-android-optimize.txt".
--keepattributes *Annotation*
-
-# keep this around for some enums that ACRA needs
--keep class org.acra.ReportingInteractionMode {
-    *;
-}
-
--keepnames class org.acra.sender.HttpSender$** {
-    *;
-}
-
--keepnames class org.acra.ReportField {
-    *;
-}
-
-# keep this otherwise it is removed by ProGuard
--keep public class org.acra.ErrorReporter
-{
-    public void addCustomData(java.lang.String,java.lang.String);
-    public void putCustomData(java.lang.String,java.lang.String);
-    public void removeCustomData(java.lang.String);
+-keepclasseswithmembernames class * {
+    @butterknife.* <fields>;
 }
-
-# keep this otherwise it is removed by ProGuard
--keep public class org.acra.ErrorReporter
-{
-    public void handleSilentException(java.lang.Throwable);
+-keepclasseswithmembernames class * {
+    @butterknife.* <methods>;
 }
 
-# GMS
+#GMS
```

Comment:
Do you think that we should use separate files for every library project we have? Where should we keep those files?